cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3816,7 +3816,7 @@ request.ui_dispatch_view = search
 search = | inputlookup cs_all_vuln where  (status="open" OR status="reopened") \
 | dedup vul_id \
 | eval _time=first_found \
-| eval days_ago=relative_time(now(), "-4h@m") \
+| eval days_ago=relative_time(now(), "-4h@h") \
```

Comment:
Why @h?

Reply:
this alert run after 5 min of lookup gen alert. To cover that 5 min. kept it to @h

Reply:
Make sense!!!